services/cloudflareService.js

```diff
@@ -0,0 +1,23 @@
+const { fetch } = require("../utils/fetch");
+
+const CLOUDFLARE_ZONE_ID = config.get("cloudflare.CLOUDFLARE_ZONE_ID");
+const CLOUDFLARE_PURGE_CACHE_API = `https://api.cloudflare.com/client/v4/zones/${CLOUDFLARE_ZONE_ID}/purge_cache`;
```

Comment:
Cloudflare itself provides this, cant it be in the constants file? let's say v4 becomes v5 we will have to search and change it, keeping it in the constants file would make it easier.

Reply:
Yeah sure will make the change